test/test_csaf.py

```diff
@@ -246,35 +270,49 @@ def test_format_references():
     assert ids == [
         {"system_name": "Red Hat Bugzilla ID", "text": "2224245"},
         {
-            "system_name": "GitHub Issue [FasterXML/jackson-databind]",
+            "system_name": "GitHub Issue [fasterxml/jackson-databind]",
```

Comment:
Can we retain the same case for system_name?

Reply:
This is related to the other comment, since I am extracting the repo from the url. I will switch to using the re.IGNORECASE flag instead of doing .lower()